test/geocoders/nominatim.py

```diff
@@ -341,6 +341,18 @@ def test_namedetails(self):
         )
         self.assertNotIn('namedetails', result.raw)
 
+    def test_reverse_zoom_parameter(self):
+        query = "33.8120962, -117.9211682"
+        result_reverse = self.reverse_run(
+            {"query": query, "exactly_one": True, "zoom": "10"},
+            {},
+        )
+
+        self.assertEqual(
+            result_reverse.raw["display_name"],
+            "Anaheim, Orange County, California, United States of America"
+        )
```

Comment:
Could you please add to this test the same query but without `zoom`, which would demonstrate that the `zoom` parameter actually made the response different?

Reply:
Done.